### PR TITLE
DMS validation and support level

### PR DESCRIPTION
**Pull request by taroface**
DOC-6170

- Add a step for data validation in DMS.
- Update DMS support level on the third-party tooling page to `Full` (v22.2 and v23.1).

### CODE REVIEW

File: v22.2/aws-dms.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ A database migration task, also known as a replication task, controls what data
 1. For the **Editing mode** radio button, keep **Wizard** selected.
 1. For the **Target table preparation mode**, select **Truncate**, **Drop tables on target**, or **Do nothing**.
     <img src="{{ 'images/v22.2/aws-dms-task-settings.png' | relative_url }}" alt="AWS-DMS-Task-Settings" style="max-width:100%" />
+1. Check **Enable validation** to compare the data in the source and target rows, and verify that the migration succeeded. You can view the results in the [**Table statistics**](#step-3-verify-the-migration) for your migration task. For more information about data validation, see the [AWS documentation](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Validating.html).
```
Comment (otan) [inline v22.2/aws-dms.md:116]: probably work marking this as optional.

**Review by otan — APPROVED** [review]

File: v22.2/aws-dms.md (hunk 1 above)
Comment (taroface) [reply v22.2/aws-dms.md]: Done.

**Review by otan — APPROVED** [review]

**Review by kathancox — APPROVED** [review]
LGTM!

**Comment by awoods187** [comment]
Awesome--thank you

**Pull request merged**